google/cloud/storage/bucket_metadata.h

```diff
@@ -819,8 +833,15 @@ class BucketMetadata : private internal::CommonMetadata<BucketMetadata> {
     return *this;
   }
 
+  /// Returns the location type (e.g. regional vs. dual region).
   std::string const& location_type() const { return location_type_; }
 
```

Comment:
Nit: Similar blocks omit this separator line.

Reply:
Fixed (at least for new functions)